.gitignore

```diff
@@ -15,4 +15,8 @@
 # vendor/
 testfiles/.DS_Store
 .DS_Store
-.vscode
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
@h0x0er can you please fix the .gitignore file?

Reply:
@varunsh-coder I fixed the issue and also added a new action-security.yml